docs/installation/gke.md

```diff
@@ -8,12 +8,15 @@ There are 3 critical quotas you will need to verify for this guide. The minimum
 
 | Quota                      | Location      | Min Value |
 |----------------------------|---------------|-----------|
+| Preemptible TPU v5 Lite Podslice chips | `<your-region>` | 8 |
```

Comment:
We should call out somewhere that you really only need TPU or GPU quota depending on what you are trying to do.

Reply:
good call, I updated the description above the table